starter-code/99-bottles-hw/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title> 99 Bottles of Beer </title>
+	</head>
+	<body>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+		<script>
+		$("body").append("<ul>");
+
+		for (Counter = 99; Counter >= 1; Counter--){
+
+		$("ul").append("<li>" + Counter  + " bottles of beer on the wall, " + Counter  + " bottles of beer. Take one down and pass it around, " + (Counter -1) + " bottles of beer on the wall." + "<br/></li>");
```

Comment:
The code doesn't handle both singular and plural cases because you didn't use `conditionals` as you have done in (03-conditionals-and-loops) homework.